timesketch/api/v1/resources/scenarios.py

```diff
@@ -684,6 +684,18 @@ def get(self, sketch_id, question_id):
         """
         sketch = Sketch.get_with_acl(sketch_id)
         question = InvestigativeQuestion.get_by_id(question_id)
+        for conclusion in question.conclusions:
+            conclusion.conclusion_events = []
+            for event in conclusion.events:
+                query_string = f"_id:{event.document_id}"
```

Comment:
This will run 1 search query per event. But you could also construct a `query_string` like this: `f"_id:({event-1.document_id} OR {event-2.document_id} OR ...)"` and run the search just once.

Reply:
Done.